manual/manual/cmds/intf-c.etex

```diff
@@ -2278,6 +2278,101 @@ names, declared in "<caml/signals.h>":
 Intuition: a ``blocking section'' is a piece of C code that does not
 use the OCaml run-time system, typically a blocking input/output operation.
 
+\section{Advanced topic: interfacing with Windows Unicode APIs}
+\label{s:interfacing-windows-unicode-apis}
+
+This section contains some general guidelines for writing C stubs that use
+Windows Unicode APIs. It also shows how to write such stubs so that they can be
+compiled under Unix (where typically nothing special needs to be done to support
+Unicode).  This is useful when binding functions that exist in both platforms.
+
+The Windows API uses the UTF-16 encoding to support Unicode. The OCaml runtime
+performs the necessary conversions so that OCaml code only needs to deal with
+UTF-8 strings. C stubs that call Windows Unicode APIs need to use specific
+runtime functions to perform the necessary conversions in a compatible way.
+
+The native C character type under Windows is "WCHAR", two bytes wide, while
+under Unix it is "char", one byte wide. A type "char_os" is defined in
+"<caml/misc.h>" that stands for the concrete C character type of each platform.
+
+In what follows, we say that a string has the \emph{native OCaml encoding} if it
+is UTF-8-encoded under Windows and an arbitrary string under Unix.
```

Comment:
Perhaps mention here the two modes (in non-Unicode mode, "native OCaml encoding" means encoded in the current code page)?